src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -152,6 +152,13 @@ private function get_integration_classes( $node_data ) {
 	 * @return string
 	 */
 	public function get_string_name( $node_id, $field, $settings ) {
+		if ( 'heading' === $settings[ $this->type ] ) {
+			$header_size = isset( $settings[ $this->settings_field ]['header_size'] ) ? $settings[ $this->settings_field ]['header_size'] : '';
+			if ( $header_size ) {
+				return $field['field'] . '-' . $settings[ $this->type ] . '-' . $node_id . '-' . $header_size;
```

Comment:
@Igor-WPML I'm not sure this is the best approach. The name of the string will change if the user changes the heading size and I expect they will then loose any existing translations they had.

Reply:
@brucepearson I checked this carefully, and yes Bruce, you are right (( Working on solving the issue.

Reply:
@brucepearson  Following to Pierre's advice, I have added column to strings table. This solves the issue. Please also refer to https://github.com/OnTheGoSystems/wpml-page-builders/pull/63